### PR TITLE
feat: enable log button for terminated sessions

### PR DESCRIPTION
**Pull request by adrysn**
* Only for >= 20.03. Logging for terminated sessions is not supported <= 19.09.

### CODE REVIEW

**Comment by inureyes** [comment]
@adrysn globalThis does not fire the changed event after component initialization so we need to create local variable to match with globalThis variable. e.g. this.switch1 = globalThis.switch1 in firstUpdated() / backend-ai-connected section.

File: src/components/backend-ai-session-list.ts [hunk 1]
```diff
@@ -1100,14 +1100,15 @@ export default class BackendAiSessionList extends BackendAIPage {
                                @click="${(e) => this._openTerminateSessionDialog(e)}"
                                icon="delete"><wl-icon>delete</wl-icon></wl-button>
           ` : html``}
-          ${this._isRunning ? html`
+          ${this._isRunning || globalThis.backendaiclient.APIMajorVersion > 4 ? html`
```
Comment (inureyes) [inline src/components/backend-ai-session-list.ts]: globalThis does not fire the changed event after component initialization so we need to create local variable to match with globalThis variable. 

e.g. 
```
firstUpdate() {
...
   document.addEventListener('backend-ai-connected', () => {
      this.switch1 = globalThis.switch1;
...
}
...
${this._isRunning || this.switch1 > 4 ? html`
```

Comment (adrysn) [reply src/components/backend-ai-session-list.ts]: Would there be any case that APIMajorVersion can be changed after component initialization?

Comment (adrysn) [reply src/components/backend-ai-session-list.ts]: I registered API major version as a property.

**Pull request merged**